src/transactionComputer.ts

```diff
@@ -94,6 +95,17 @@ export class TransactionComputer {
         transaction.guardian = guardian;
     }
 
+    applyRelayer(transaction: ITransaction, relayer: Address) {
```

Comment:
In sdk-py, we don't have it:

https://github.com/multiversx/mx-sdk-py/blob/feat/next/multiversx_sdk/core/transaction_computer.py#L83

Should we have it?

https://github.com/multiversx/mx-sdk-specs/blob/main/core/transaction_computer.md

Reply:
I suggest we delete it from specs, as well. It is useless.